### PR TITLE
Fix 403 error when downloading Mnist dataset in Pytorch Lighting example

### PR DESCRIPTION
**Pull request by JanuszL**
- adds a work around for the issue described in https://github.com/pytorch/vision/issues/1938

Signed-off-by: Janusz Lisiecki <jlisiecki@nvidia.com>

#### Why we need this PR?
*Pick one, remove the rest*
- It fixes 403 error when downloading Mnist dataset in Pytorch Lighting example

#### What happened in this PR?
*Fill relevant points, put NA otherwise. Replace anything inside []*
 - What solution was applied:
     adds a work around for the issue described in https://github.com/pytorch/vision/issues/1938
 - Affected modules and functionalities:
     pytorch-lightning.ipynb
 - Key points relevant for the review:
     NA
 - Validation and testing:
     CI
 - Documentation (including examples):
     pytorch-lightning.ipynb has been updated

**JIRA TASK**: *[NA]*

### CODE REVIEW

**Comment by JanuszL** [comment]
!build

File: docs/examples/frameworks/pytorch/pytorch-lightning.ipynb [hunk 1]
```diff
@@ -33,7 +33,13 @@
...
     "import os\n",
     "\n",
-    "BATCH_SIZE = 64"
+    "BATCH_SIZE = 64\n",
+    "\n",
+    "# workaround for https://github.com/pytorch/vision/issues/1938 - error 403 when downloading mnist dataset\n",
+    "from six.moves import urllib\n",
+    "opener = urllib.request.build_opener()\n",
+    "opener.addheaders = [('User-agent', 'Mozilla/5.0')]\n",
+    "urllib.request.install_opener(opener)"
```
Comment (klecki) [inline docs/examples/frameworks/pytorch/pytorch-lightning.ipynb:42]: Does this work with just the urllib.request? https://docs.python.org/3/library/urllib.request.html#module-urllib.request

`six` is compat layer for Python 2.

Comment (JanuszL) [reply docs/examples/frameworks/pytorch/pytorch-lightning.ipynb]: Fixed.

**Comment by JanuszL** [comment]
!build

**Review by klecki — APPROVED** [review]

**Review by jantonguirao — APPROVED** [review]

**Pull request merged**